commit 41bf160d2e1efc5185d2f26431394f3feffa0ce4
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Date:   Mon Mar 19 09:24:52 2018 +0100

    data/org.ayatana.indicator.power.gschema.xml.in: Prepend project namespace to enum id.
    
    Fixes AyatanaIndicators/ayatana-indicator-power#1.

diff --git a/data/org.ayatana.indicator.power.gschema.xml.in b/data/org.ayatana.indicator.power.gschema.xml.in
index 872b78e..2119aa9 100644
--- a/data/org.ayatana.indicator.power.gschema.xml.in
+++ b/data/org.ayatana.indicator.power.gschema.xml.in
@@ -1,5 +1,5 @@
 <schemalist>
-  <enum id="icon-policy-enum">
+  <enum id="ayatana-indicator-power-icon-policy-enum">
     <value nick="present" value="0" />
     <value nick="charge" value="1" />
     <value nick="never" value="2" />
@@ -15,7 +15,7 @@
       <_summary>Show percentage in Menu Bar</_summary>
       <_description>Whether or not to show the percentage in the menu bar.</_description>
     </key>
-    <key enum="icon-policy-enum" name="icon-policy">
+    <key enum="ayatana-indicator-power-icon-policy-enum" name="icon-policy">
       <default>"present"</default>
       <_summary>When to show the battery status in the menu bar.</_summary>
       <_description>Options for when to show battery status. Valid options are "present", "charge", and "never".</_description>
